recipes/caterva/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "caterva" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Blosc/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 916387022b1bfe12268eb8362f631dc0da289b1470d941b7c2ff163172f2eb1f
+  patches:
+    # Do not try to use CXX in this project
+    - caterva.patch
+
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('caterva') }}
+  missing_dso_whitelist:
+    - /lib/libc.so.6
+    - /lib/libselinux.so.1
+    - /lib64/libcap.so.2
+    - /lib/librt.so.1
+    - /lib64/libacl.so.1
+    - /lib/ld-linux.so.2
+    - /lib/libpthread.so.0
+    - /lib64/libattr.so.1
```

Maybe using this you can get past the issues for now. 
```suggestion
    - /lib64/libattr.so.1
  ignore_run_exports:
    - c-blosc2
```